compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangAnonymousModelHelper.java

```diff
@@ -35,13 +35,15 @@
     private Map<PackageID, Integer> anonTypeCount;
     private Map<PackageID, Integer> anonServiceCount;
     private Map<PackageID, Integer> anonFunctionCount;
+    private Map<PackageID, Integer> anonForkCount;
```

Comment:
Minor change, can we just call this `forkCount` instead? Since it is not really anon?